app/global-translations/locale-es.json

```diff
@@ -831,7 +831,7 @@
 	"error.msg.client.duplicate.accountNo": "Ya existe un cliente con el número de cuenta `{{params[0].value}}`.",
 	"error.msg.client.duplicate.dpi": "Un cliente con dpi. `{{params[0].value}}` ya existe.",
 	"error.msg.client.blacklisted": "Esta persona esta en la lista negra. MOTIVO: `{{params[0].value}}` ",
-	"error.msg.client.id.invalid": "El cliente no existe.",
+	"error.msg.client.id.invalid": "No se pudieron volver a intentar los detalles del cliente con id `{{params[0].value}}`: `{{params[1].value}}`.",
```

Comment:
Please adjust it to "No se pudieron recuperar los detalles del cliente con id `{{params[0].value}}`: `{{params[1].value}}`.",